eclass/ecm.eclass

```diff
@@ -334,6 +352,53 @@ DEPEND+=" ${COMMONDEPEND}"
 RDEPEND+=" ${COMMONDEPEND}"
 unset COMMONDEPEND
 
+# @FUNCTION: _ecm_check_kde_mode
+# @DESCRIPTION:
+# Make an effort to detect KDE projects, and possibly which main product.
```

Comment:
I was ready to give you a smartass answer about how we can surely use XYZ and it turns out I can't find any such things ;)